account-kit/smart-contracts/src/light-account/clients/client.ts

```diff
@@ -31,14 +36,26 @@ export type CreateLightAccountClientParams<
   >;
 
 export function createLightAccountClient<
-  TChain extends Chain | undefined = Chain | undefined,
   TSigner extends SmartAccountSigner = SmartAccountSigner
 >(
-  args: CreateLightAccountClientParams<Transport, TChain, TSigner>
+  params: AlchemyLightAccountClientConfig<TSigner>
+): Promise<
+  AlchemySmartAccountClient<
+    Chain | undefined,
+    LightAccount<TSigner>,
+    LightAccountClientActions<TSigner>
+  >
+>;
```

Comment:
oh nice! this is way cleaner than what I was thinking which was adding to the conditional typing in for `CreateLightAccountClientParams` based on the transport

Reply:
Had to verify this, and slight variation. Good thing for tests right?